Source/CarthageKit/Xcode.swift

```diff
@@ -790,10 +804,36 @@ public func buildScheme(scheme: String, withConfiguration configuration: String,
 		var argsForBuilding = argsForLoading
 		argsForBuilding.onlyActiveArchitecture = .No
 
-		var buildScheme = xcodebuildTask("build", argsForBuilding)
-		buildScheme.workingDirectoryPath = workingDirectoryURL.path!
+		func fetchDestination() -> SignalProducer<String?, ReactiveTaskError> {
```

Comment:
Can you add a comment describing why this is necessary and perhaps linking to #417?